.env

```
@@ -0,0 +1,3 @@
+MESHY_API_KEY=msy_3ClctZx7svFvzl4kvgXqP2Sa1z7aNIu74dwf
+JULES_API_KEY=AQ.Ab8RN6KwhjKaLXoz4gtOcVKeCiiXfSW-z0czzA4VBR70Y3MgWQ
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JULES_API_KEY key should go before the MESHY_API_KEY key

(UnorderedKey)